emerlin2caom2/casa_reader.py

```diff
@@ -0,0 +1,46 @@
+import numpy
+
+import casatools
+
+msmd = casatools.msmetadata()
```

Comment:
In main_app.py, msmd is .measurement_set_metadata, whereas here msmd is the casa msmetadata; potential confusion for package readability.